src/Farmer/Common.fs

```diff
@@ -474,6 +479,26 @@ module CognitiveServices =
         | TextAnalytics
         | TextTranslation
 
+    type CognitiveServicesSku =
```

Comment:
I wonder whether we really need this validation? Perhaps we can encode this rule within the type system in a similar way that we have done with Storage accounts.

Reply:
In fact, if Bing Search is to be removed anyway (per #526) then we can remove this entire validation.